### ISSUE TITLE
ID has `draft`, but type is `standard`

### ISSUE THREAD

**Issue by strogonoff** [ietf-ribose/relaton-data-ids#8]
Wikipedia [describes](https://en.wikipedia.org/wiki/Internet_Draft) Internet Drafts in a way that strongly hints we should use document type `draft`, not `standard`. For example, it claims they are not supposed to be relied on by the public at large.

From the article I get the feeling that RFC is a standard, but I-D is not. Unless I am mistaken as to what “type” is supposed to mean in Relaton?

However, we use `standard` for I-Ds: https://github.com/ietf-ribose/relaton-data-ids/blob/main/data/draft-donley-behave-deterministic-cgn-02.yaml#L13

**Comment by strogonoff** [comment]
Also, there is an attribute [`draft`](https://github.com/relaton/relaton-models/blob/main/grammars/biblio.rnc#L457), which I don’t know how to tie into this.

**Comment by larseggert** [comment]
I-Ds are **definitely not** standards.

**Comment by ronaldtse** [comment]
@strogonoff I prefer not to involve the Relaton model here but this is a good point.

We know I-Ds are drafts. But should we consider them "drafts on path to RFCs"?

P.S. ISO 690/Relaton has a long list of accepted types that are not "standards":
![](https://github.com/relaton/relaton-models/raw/main/images/RelBib_BibItemType.png)

**Comment by larseggert** [comment]
> We know I-Ds are drafts. But should we consider them "drafts on path to RFCs"?

No, you should not. Quite a few I-Ds are not meant to be ever published as RFCs.

**Comment by ronaldtse** [comment]
Thanks @larseggert , that’s what I suspected. 

If we consider I-Ds as “tech reports”, would that be a reasonable match?

**Comment by larseggert** [comment]
That's what the bibtex citations in the datatracker generate at the moment, so I think that should be fine.

**Comment by ronaldtse** [comment]
Thanks @larseggert , somehow I missed this message. We should change the `type` to "tech report" instead.

**Comment by rjsparks** [comment]
@ronaldtse - is there anything that still needs to be done for this issue, or can it be closed?
